src/client/styles/skeleton.scss

```diff
@@ -0,0 +1,68 @@
+/**
+ *
+ *  Online store PWA sample.
+ *  Copyright 2017 Google Inc. All rights reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ *
+ */
+h1.skeleton::before {
```

Comment:
Any particular reason why you added the `h1`? I'd prefer not to lock the skeleton functionality to a particular element.

Reply:
It's because height of the skeleton is hardcoded, so this one is suitable only for this type of header.

Reply:
So you're relying on the browser's default styles, I take it? I'd prefer for us to have a `.skeleton {` selector instead with explicit sizes / font sizes instead, so it could be used with any `<span>` or `<div>`, not just `<h1>`s.

This isn't a huge priority or anything, though. We can leave this for the inevitable future pass on CSS we'll have once we decide to standardise our styles across the project :)

Reply:
I'm very open to experiment on this. I think we'd want the "skeleton" to look differently on a div than h1 though (e.g. few lines of dummy text vs one line). I'll add a TODO here to remember this discussion in further development.